src/main/java/com/imgix/URLHelper.java

```diff
@@ -14,6 +15,8 @@
 
 public class URLHelper {
 
+  private static final String IS_ENCODED = "isEncoded";
```

Comment:
Do these values _need_ to be `String`? The string value `"isEncoded"` is the same as its identifier `IS_ENCODED`, what if `... Boolean IS_ENCODED = false`?

Reply:
Linter was yelling at me for not storing these as a constant 🤷🏼‍♂️ , but I can change this around if you like.

Reply:
My 2¢ here, I agree with @ericdeansanchez